packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -178,8 +178,13 @@
     }
 
     async function sendOutputAndClosePopup(): Promise<void> {
-        await sendOutput(preparedOutput)
-        closePopup()
+        try {
+            await sendOutput(preparedOutput)
+        } catch (err) {
+            handleError(err)
+        } finally {
+            closePopup()
```

Comment:
If there was an error we should probably not close the popup, if before this gave an exception the popup was kept open, i would probably keep the same logic, or did you change it for a reason?

Reply:
When I reject a transaction from the ledger I receive an error, also this popup cannot be closed. If I dont close the popup after the error, the user will have the screen locked and unable to do anything in the ledger 🥲 
Whether you accept or reject the transaction from the ledger, I think it is good to close the popup